docs/user-guide/node-selection/index.md

```diff
@@ -198,3 +198,194 @@ must be satisfied for the pod to schedule onto a node.
+hardware off of those nodes, so as to leave room for later-arriving pods that do need the
+specialized hardware. This can be done by tainting the nodes that have the specialized
+hardware (e.g. `kubectl taint nodes nodename special=true:NoSchedule` or
+`kubectl taint nodes nodename special=true:PreferNoSchedule`) and adding a corresponding
+toleration to pods that use the special hardware. As in the dedicated nodes use case,
+it is probably easiest to apply the tolerations using a custom
+[admission controller](https://kubernetes.io/docs/admin/admission-controllers/)) that
+uses some characteristic(s) of the pod to determine that it should be allowed to use
+the special nodes and hence add the toleration. To ensure that the pods that need
+the special hardware *only* schedule onto the nodes that have the special hardware, you will need some
+additional mechanism, e.g. you could represent the special resource using
+[opaque integer resources](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#opaque-integer-resources-alpha-feature)
+and request it as a resource in the PodSpec, or you could label the nodes that have
+the special hardware and use node affinity on the pods that need the hardware.
+
+* **per-pod-configurable eviction behavior when there are node problems (alpha feature)**,
+which is described in the next section.
+
+### Per-pod-configurable eviction behavior when there are node problems (alpha feature)
+
+Earlier we mentioned the `NoExecute` taint effect, which affects pods that are already
+running on the node as follows
+
+ * pods that do not tolerate the taint are evicted immediately
+ * pods that tolerate the taint without specifying `tolerationSeconds` in
+   their toleration specification remain bound forever
+ * pods that toleate the taint with a specified `tolerationSeconds` remain
+   bound for the specified amount of time
+
+The above behavior is a beta feature. In addition, Kubernetes 1.6 has alpha
+support for representing node problems (currently only "node unreachable" and
+"node not ready", corresponding to the NodeCondition "Ready" being "Unknown" or
+"False" respectively) as taints. When the `TaintBasedEvictions` alpha feature
+is enabled (`--feature-gates=TaintBasedEvictions`), pods can request to stay bound to a node that has one
+or both of these problems, for a per-pod-configurable amount of time.
+For example, an application with a lot of local state might want to stay
+bound to node for a long time in the case of network partition, in the hope
+that the partition will recover and thus the pod eviction can be avoided.
+The toleration the pod would use in that case would look like
+
+```
+tolerations: 
+   - key: "node.alpha.kubernetes.io/unreachable"
+     operator: "Exists"
+     effect: "NoExecute"
+     tolerationSeconds: 6000
+```
+
+(The key for node not ready is `node.alpha.kubernetes.io/notReady`.)
```

Comment:
How about update this a bit as:

For `node not ready` case, just update the `key` for `tolerations` to `node.alpha.kubernetes.io/notReady`.

Also do you want to add sth for daemonset (https://kubernetes.io/docs/admin/daemons/ ) here? I'm planning to update the daemonset a bit and perhaps we can add a link here, I can follow up with a PR once this got merged.